### ISSUE TITLE
Next line key

### ISSUE THREAD

**Issue by markvdw** [rmcgibbo/jupyter_emacskeys#3]
First off, I love this, thanks a lot!

The "ctrl+n" shortcut doesn't work for me though, while "ctrl+p" does. I'm using Firefox both with and without keybindings for all GTK text boxes.

Any suggestions?

**Comment by rmcgibbo** [comment]
Sorry, I have no idea.

**Comment by cipri-tom** [comment]
I believe that `ctrl+n` is eaten up by Firefox, as it's the shortcut for "new window"

**Comment by aiguofer** [comment]
That's correct, had a chat about this at https://github.com/jupyter/notebook/issues/1794

**Comment by yelled1** [comment]
I love this prj, as well. Along with my blue switch keyboards. Actually [https://addons.mozilla.org/en-US/firefox/addon/keybinder/?src=search](url) works! Which means Ctrl-n can be disabled for Firefox. Ctrl-Shift-n works. A little annoying but it's a work around for now.
If this is NOT too much of burden pls fix. Thanks much,
